onnx_xla/onnx_xla_interface.cc

```diff
@@ -208,7 +210,8 @@ ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI ONNXIFI_SYMBOL_NAME(
       return ONNXIFI_STATUS_INVALID_SIZE;
     }
 
-    *graph = (onnxGraph) backendcontroller->build(onnxModel, onnxModelSize);
+    *graph = (onnxGraph) backendcontroller->build(onnxModel, onnxModelSize,
```

Comment:
`*graph` must be set to `NULL`/`nullptr` if the function fails. I suggest to do it in the first line of the function, and then overwrite null value if function succeeds.

Reply:
added *graph = NULL to beginning of try block